client/state/analytics/reducer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+
+export const isTracking = ( state = false, { type } ) => {
```

Comment:
I think we should rename the reducer in order to not confuse it with the selector, to something that doesn't sound like a getter or predicate. Since this is the top-level reducer.js file in `analytics`, it should actually export a reducer that is imported as `analytics` from `state/index.js`. So maybe I need to reconsider, and we should make this a subreducer...

I'd suggest to name this one `tracking`, and `export default combinedReducers( { tracking } );`.